lib/server.rb

```diff
@@ -6,6 +6,15 @@ class Server < Grape::API
   format :json
   prefix :api
 
+  rescue_from Grape::Exceptions::ValidationErrors do |e|
```

Comment:
this class is too long for the linting. I'll add this to the rubocop linting to ignore